CMakeLists.txt

```diff
@@ -514,6 +514,10 @@ endif (STATIC_LINK)
 
 INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR} gemrb/includes gemrb/core platforms/${CMAKE_SYSTEM_NAME})
 
+if (NOT DEFINED COLOR_LOG)
```

Comment:
I suggest you add NOCOLOR as well, for anyone requesting it from before. Another block to handle the transition.

Reply:
Done

Reply:
Should be false though, since you inverted the meaning of the var.

Reply:
yeah, I spotted that as soon as I hit enter 🤦🏻‍♂️